test/frameworks.js

```diff
@@ -74,3 +81,13 @@ each(FRAMEWORKS, (info, framework) => {
     t.is(validate(framework, FRAMEWORK_JSON_SCHEMA), true)
   })
 })
+
+test('each json file should be required in main.js FRAMEWORKS', async (t) => {
+  const dir = path.join(__dirname, '..', 'src', 'frameworks')
```

Comment:
[sand] In most cases, using forward slashes work on Windows, because almost all the Node.js API methods (like `fs.readdir()`) handle. Usually, `path.join()` only matters when the file path is output to a file or a terminal, or is a binary executed in the terminal. [See some write-up on this](https://github.com/ehmicky/cross-platform-node-guide/blob/main/docs/3_filesystem/file_paths.md#nodejs-behavior)

Reply:
Thanks, fixed in https://github.com/netlify/framework-info/pull/57/commits/a6084a8ead3c00a500d2b95edb745877d6f84882